src/pymor/analyticalproblems/expressions.py

```diff
@@ -269,6 +296,10 @@ def __init__(self, value):
         self.value = value
         self.numpy_symbol = repr(value)
 
+    def fenics_expr(self, params):
+        from dolfin import Constant
+        return np.vectorize(Constant)(self.value)
```

Comment:
this should work, but

```suggestion
        return np.array(Constant(self.value))
```

is probably more idiomatic now that `Constant.shape` is forced to `()`.